x/poe/contract/tgrade_validator_voting_integration_test.go

```diff
@@ -74,6 +86,64 @@ func TestValidatorsGovProposal(t *testing.T) {
 				assert.Equal(t, exp, gotPlan)
 			},
 		},
+		"cancel upgrade": {
+			src: contract.ValidatorProposal{
+				CancelUpgrade: &struct{}{},
+			},
+			assertExp: func(t *testing.T, ctx sdk.Context) {
+				_, exists := example.UpgradeKeeper.GetUpgradePlan(ctx)
+				assert.False(t, exists, "does not exist")
+			},
+		},
+		"update block params": {
+			src: contract.ValidatorProposal{
+				UpdateConsensusBlockParams: &contract.ConsensusBlockParamsUpdate{
+					MaxBytes: 10000000,
+					MaxGas:   20000000,
+				},
+			},
+			assertExp: func(t *testing.T, ctx sdk.Context) {
+				consensusParams := example.BaseApp.GetConsensusParams(ctx)
+				var expConsensusParams = app.DefaultConsensusParams
```

Comment:
When you test against default values it is hard to tell if the proposal did anything. Or were the values modified before?

Reply:
They were modified. I'll switch the order here, for clarity